databases/week3/sqlHW.sql

```diff
@@ -0,0 +1,246 @@
+CREATE DATABASE Homework3DB;
+
+USE Homework3DB;
+
+CREATE TABLE Meal (
+    `id` INT NOT NULL AUTO_INCREMENT,
+    `title` VARCHAR(255),
+    `description` TEXT,
+    `location` VARCHAR(255),
+    `when` DATETIME,
+    `maxreservation` INT,
+    `price` DECIMAL(10, 2),
+    `created_date` DATE,
+    PRIMARY KEY(`id`)
+) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+
+CREATE TABLE Reservation(
+    `id` INT NOT NULL AUTO_INCREMENT,
+    `number_of_guests` INT,
+    `meal_id` INT,
+    `created_date` DATE,
+    `contact_phone_number` VARCHAR(255),
+    `contact_name` VARCHAR(255),
+    `contact_email` VARCHAR(255),
+    PRIMARY KEY (`id`),
+    FOREIGN KEY (meal_id) REFERENCES `Meal` (`id`)
+) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+
+CREATE TABLE Review(
+    `id` INT NOT NULL AUTO_INCREMENT,
+    `title` VARCHAR(255),
+    `description` TEXT,
+    `meal_id` INT,
+    `stars` INT,
+    `created_date` DATE,
+    PRIMARY KEY (`id`),
+    FOREIGN KEY (meal_id) REFERENCES `Meal` (`id`)
+) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
```

Comment:
I think at least some of the values should be NOT NULL

Reply:
Agree, I think I can do NOT NULL all of them excluding description